detectors/resources/src/main/java/com.google.cloud.opentelemetry.detectors/GCEResource.java

```diff
@@ -0,0 +1,63 @@
+package com.google.cloud.opentelemetry.detectors;
```

Comment:
For some reason, you have a directory named `com.google.cloud.opentelemetry.detectors` containing this file.
Instead, in Java it should be a directory `com` with a subdirectory `google` and so on.
Please fix.